hack/pre-release.sh

```diff
@@ -21,36 +23,48 @@ usage() {
   echo "  bumps the baseimage versions"
 }
 
-# This function updates the WMCO version in all relevant files 
+# This function increments any xyz version given an x y or z 
 # takes 2 arguments 
-# 1st arg: stream to update. Can be x, y or z to represent the x y or z stream. 
-# 2nd arg: WMCO version to update.
-update_WMCO_version() { 
+# 1st arg: version to update.
+# 2nd arg: stream to update. Can be x, y or z to represent the x y or z stream. 
+increment_version() {
   if [ "$#" -ne 2 ]; then
-    echo incorrect parameter count for update_WMCO_version $#
+    echo incorrect parameter count for increment_version $#
     return 1
   fi
-
-  xyz_stream=$1
-  version=$2 
-  cluster_service_version_yaml=config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml
-
+  version=$1
   # Setting the stream var to 1 2 or 3 which is used for incrementing the correct stream
   # this tells awk which column to act on. Column 1, column 2, or column 3.
-  # which correspond to the x, y and z stream respectively. 
-  if [[ $xyz_stream == "x" ]]; then
+  # which correspond to the x, y and z stream respectively.
+  if [[ $2 == "x" ]]; then
     stream_var=1
   fi
-  if [[ $xyz_stream == "y" ]]; then
+  if [[ $2 == "y" ]]; then
     stream_var=2
   fi
-  if [[ $xyz_stream == "z" ]]; then
+  if [[ $2 == "z" ]]; then
     stream_var=3
   fi
- 
+
   # Increment the version
-  updated_version=$(echo $version |
-    awk -F. -v OFS=. -v stream_var="$stream_var" '{$stream_var += 1 ; print}')
+  echo $version | awk -F. -v OFS=. -v stream_var="$stream_var" '{$stream_var += 1 ; print}'
```

Comment:
I had asked for this during the first review also. Please add a comment about what you are doing here. Splitting on `.` etc.